tsg

```diff
@@ -0,0 +1,80 @@
+(module) @this
```

Comment:
Can we put some comments (tsg supports comments, right?) that indicate what this file is for, should someone stumble across it?